discovery_engine_core/core/src/document.rs

```diff
@@ -282,6 +282,19 @@ mod tests {
         }
     }
 
+    impl TryFrom<Article> for HistoricDocument {
```

Comment:
If we need  it make tests simpler we can define it in test module instead.

Reply:
not sure what you mean? this impl is already in a test module.

Reply:
🙈 I'm blind sorry.